docker-compose.yml

```diff
@@ -41,7 +41,8 @@ services:
     volumes:
       - datadir:/etc/georchestra
     environment:
-      - JAVA_TOOL_OPTIONS=-Dgeorchestra.datadir=/etc/georchestra -Dspring.profiles.active=docker -Xmx512M -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=0.0.0.0:5005
+      - SPRING_PROFILES_ACTIVE=docker,json-logs
```

Comment:
using a separate environment variable here might mean some changes on the georchestra helm chart (ping @edevosc2c). We will still have the possibility to pass the profiles in the JAVA_TOOL_OPTIONS though.

Reply:
Right, this change only affects the development docker-compose files, which aren't used in production deployments.

The Helm charts remain unaffected since they're separate deployment configurations. If we want to add the json-logs profile to production deployments via Helm, we could do that either through JAVA_TOOL_OPTIONS or SPRING_PROFILES_ACTIVE, whichever is preferred.

This change just makes the development environment use structured JSON logging by default, which is helpful for local testing.

Reply:
yup, I had in mind a flag in the yaml from the helm chart like `jsonlogs: true` which would actually append the json-logs profile to the environment variable or so.